lib/mpl_toolkits/tests/test_axisartist_grid_finder.py

```diff
@@ -1,6 +1,27 @@
+from matplotlib.transforms import Bbox
 from mpl_toolkits.axisartist.grid_finder import (
-    FormatterPrettyPrint,
-    MaxNLocator)
+    _find_line_box_crossings, FormatterPrettyPrint, MaxNLocator)
+
+import numpy as np
+import pytest
```

Comment:
Third party imports go above `matplotlib`.

Reply:
yes (to both)